src/features/movies/MoviePage/Backdrop/index.js

```diff
@@ -1,36 +1,33 @@
 import {
-  BackdropContainer,
-  Details,
-  MaxRates,
-  Rates,
+  BlureFrames,
+  Container,
+  Content,
+  Rate,
   RatingContainer,
   StarIcon,
   Title,
+  Vote,
   Votes,
   Wrapper,
 } from "./styled";
-import { StyledBackdrop } from "./styled";
+import { BackdropImg } from "./styled";
 
 export const Backdrop = () => {
   return (
     <Wrapper>
-      <BackdropContainer>
-        <Details>
+      <Container>
+        <BlureFrames />
+        <BackdropImg src={"https://image.tmdb.org/t/p/w1280/bQXAqRx2Fgc46uCVWgoPz5L5Dtr.jpg"} alt="" />
```

Comment:
Z uwagi na to, że pobieramy już dane z API w src będziemy podawać już inną ścieżkę url :), alterantywny tekst również możemy przekazać jako np tytuł filmu